rspace/src/test/scala/coop/rchain/rspace/history/HistoryRepositoryGenerativeSpec.scala

```diff
@@ -57,13 +60,17 @@ class LMDBHistoryRepositoryGenerativeSpec
       emptyHistory     = HistoryInstances.merging(History.emptyRootHash, historyStore)
       exporter         = RSpaceExporterImpl[Task](historyLmdbStore, coldLmdbStore, rootsLmdbStore)
       importer         = RSpaceImporterImpl[Task](historyLmdbStore, coldLmdbStore, rootsLmdbStore)
+      channelLMDBStore <- StoreInstances.lmdbStore[Task](lmdbConfig)
+      channelStore     = new ChannelStoreImpl(channelLMDBStore, stringSerialize, codecString)
```

Comment:
To get instance of KV typed store, in tests `InMemoryStoreManager` can be used.

Reply:
see https://github.com/rchain/rchain/pull/3204#discussion_r532149581